src/stores/gameList.js

```diff
@@ -53,13 +54,15 @@ export const useGameListStore = defineStore('gameList', {
       const updatedGame = this.openGames.find((game) => game.id === data.gameId);
       if (updatedGame) {
         updatedGame.numPlayers = Math.min(2, updatedGame.numPlayers + 1);
+        updatedGame.players.push(data.newPlayer);
         updatedGame.status = data.newStatus;
       }
     },
     otherLeftGame(gameId) {
       const updatedGame = this.openGames.find((game) => game.id === gameId);
       if (updatedGame) {
         updatedGame.numPlayers = Math.max(0, updatedGame.numPlayers - 1);
+        updatedGame.players.pop();
```

Comment:
This is actually a little trickier than you have it here and will require a full stack change. Specifically, we don't know which player left, so we can't just assume it was the one at the end of the array. We need to update
```
api/controllers/game/leave-lobby.js
```

So that its `sails.sockets.blast('leftGame', { id: values[0].id }, req);` call additionally specifies _which user id left_ and then we need to update this `otherLeftGame` method to look up which specific user left the game and remove that one. It could look like this

```
    otherLeftGame(gameId, userId) {
      const updatedGame = this.openGames.find((game) => game.id === gameId);
      if (updatedGame) {
        updatedGame.players. = updatedGame.players.filter((player) => player.id !== userId);
      }
  }
```